configs/platforms/redhatfips-7-x86_64.rb

```diff
@@ -14,7 +14,6 @@
     "libsepol-devel",
     "libselinux-devel",
     "make",
-    "openssl-devel",
```

Comment:
this is a system package, so we do want it -- this is what's replacing our own custom-built ssl